edkrepo/commands/list_repos_command.py

```diff
@@ -158,16 +184,19 @@ def run_command(self, args, config):
                             branches.add(source.branch)
 
             #Sort the branch names so they will be displayed alphabetically
-            #with the exception that if a branch named "master" exists, then it
-            #will be displayed first
+            #with the exception that if a branch named "main" or "master" exists,
+            #then it will be displayed first

```

Comment:
I think it would make sense to update this comment to if main and/or master are present they will be displayed first. It does not look like you will only display one of them.